config/spryk/templates/Spryker/Glue/BackendApi/GlueBackendApiBootstrap.php.twig

```diff
@@ -8,7 +8,7 @@
 namespace {{ organization }}\Glue\GlueApplication\Bootstrap;
 
 use Spryker\Glue\GlueApplication\Bootstrap\GlueBootstrap;
-use Spryker\Glue\GlueBackendApiApplication\Plugin\GlueApplication\BackendApiGlueApplicationBootstrapPlugin;
+use Spryker\Glue\GlueApiApplication\Plugin\GlueApplication\BackendApiGlueApplicationBootstrapPlugin;
```

Comment:
Same, there is no namespace like this.

Reply:
The actual name of the class still contains "Backend", is this correct? Is there a separate storefront variation of this templete?